src/include/module/audio/audio_stream.h

```diff
@@ -20,6 +20,7 @@
  */
 struct sof_audio_stream_params {
 	uint32_t id;
+	uint32_t pipeline_id;
```

Comment:
sorry I missed it - would be nice to have dedicated procedures sink_get_pipeline_id(sink *) and source_get_pipeline_id(source *) in sink_api.h and source_api.h